docs/ExposePortsToInternet.md

```diff
@@ -6,41 +6,48 @@
 > You are viewing the calico-containers documentation for release **release**.
 <!--- end of master only -->
 
-# Expose Container Port to Host-Interface / Internet
+# Expose Container Port to Host Interface / Internet
 
 In the [Calico without Docker networking tutorial](calico-with-docker/without-docker-networking/README.md) 
 or the [Calico as a Docker network plugin tutorial](calico-with-docker/docker-network-plugin/README.md)
 we created containers and assigned endpoints (a container interface) to them. This is used for Container-
 To-Container communication.
 
-The example below shows how to expose a port of a container to the Host-Interface so this container is 
+The example below shows how to expose a port of a container to the host interface so this container is 
 reachable from outside / the internet.
 
 ## Why isn't the `-p` flag on `docker run` working as expected?
-If you connect containers to the docker0 interface, then calico would not be able to enforce security rules
-between workloads; all containers on the bridge would be able to communicate with each other.
+If you connect containers to the `docker0` bridge interface, Calico would not 
```

Comment:
I think it's worth noting.  Just tested.  I could see people running this and thinking it is working with calico policy enforcement when really it's not secured.